packages/why3/why3.1.7.0/opam

```diff
@@ -0,0 +1,90 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+authors: [
+  "François Bobot"
+  "Jean-Christophe Filliâtre"
+  "Claude Marché"
+  "Guillaume Melquiond"
+  "Andrei Paskevich"
+]
+
+homepage: "http://why3.lri.fr/"
+license: "LGPL-2.1-only"
+doc: "http://why3.lri.fr/doc/"
+bug-reports: "https://gitlab.inria.fr/why3/why3/issues"
+dev-repo: "git+https://gitlab.inria.fr/why3/why3.git"
+
+tags: [
+  "deductive"
+  "program verification"
+  "formal specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+]
+
+build: [
+  ["./autogen.sh"] {dev} # when pinning, there might be no configure file
+  ["touch" "configure"]
+  ["./configure"
+   "--prefix" prefix
+   "--disable-frama-c"
+   "--disable-coq-libs"
+   "--disable-js-of-ocaml"
+   "--disable-re"
+   "--enable-ocamlfind"
+   "--disable-zarith" {!zarith:installed}
+   "--enable-zarith" {zarith:installed}
+   "--disable-mpfr" {!mlmpfr:installed}
+   "--enable-mpfr" {mlmpfr:installed}
+   "--disable-zip" {!camlzip:installed}
+   "--enable-zip" {camlzip:installed}
+   "--disable-hypothesis-selection" {!ocamlgraph:installed}
+   "--enable-hypothesis-selection" {ocamlgraph:installed}
+   "--disable-stackify" {!ocamlgraph:installed}
+   "--enable-stackify" {ocamlgraph:installed}
+   "--disable-ide"]
+  [make "-j%{jobs}%" "all" "opt" "byte"]
+  [make "doc" "stdlibdoc" "apidoc"] {with-doc}
+]
+
+install: [
+  [make "install" "install-lib"]
+  [make "DOCDIR=%{_:doc}%" "install-doc"] {with-doc}
+]
+
+depends: [
+  "conf-autoconf" {build & dev}
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind" {build}
+  "menhir" {>= "20170418"}
+  "num"
+]
+
+depopts: [
+  "zarith"
+  "camlzip"
+  "ocamlgraph"
+  "sexplib"
+  "ppx_deriving" {build}
+  "ppx_sexp_conv" {build}
+  "mlmpfr"
+]
+
+conflicts: [
+  "why3-base"
+  "ocamlgraph" {< "1.8.2"}
+  "mlmpfr" {< "4.0.0"}
+  "base-effects"
+]
+
```

Comment:
It seems that cygwin patch is remove 1.6.0 → 1.7.0.

Is it planned to omit it for now and add it back after https://github.com/ocaml/opam-repository/pull/24841 getting merged?